Prox/Prox/Widgets/ReviewScoreView.swift

```diff
@@ -62,34 +57,58 @@ class ReviewScoreView: UIStackView {
         }
     }
 
+    private func updateReviewScore(score: Float) {
+        for (index, subview) in arrangedSubviews.enumerated() {
+            if let scoreItem = subview as? ReviewScoreItemView {
+                let scoreLeft = self.score - Float(index)
+                setFillScore(score: scoreLeft, forView: scoreItem)
+            }
+        }
+    }
+
+    private func setFillScore(score: Float, forView view: ReviewScoreItemView) {
+        if score >= 1 {
+            view.leftColor = color
+            view.rightColor = color
+        } else if score > 0 {
+            view.leftColor = color
+            view.rightColor = Colors.reviewScoreDefault
+        } else {
+            view.leftColor = Colors.reviewScoreDefault
+            view.rightColor = Colors.reviewScoreDefault
+        }
+    }
+
 }
 
 private class ReviewScoreItemView: UIView {
 
-    var color: UIColor? {
+    var leftColor: UIColor? {
```

Comment:
It might be slightly safer to have:

```
private enum FillAmount {
  case empty, half, filled
}

private class ReviewScoreItemView: UIView {
  let color: UIColor // defined in init
  var fillAmount: FillAmount = .empty // set from `updateReviewScore`/`setFillScore`
...
```
But if it already works, it's probably not worth the time.